app/controllers/case_workers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CaseWorkers::ApplicationController < ApplicationController
-  before_action :authenticate_case_worker!
+  #before_action :authenticate_case_worker!
```

Comment:
why is this commented out ? if you need to remove it remove it. else keep it and override authentication in the api controller. While we are here you might as well add basic auth with a single username/password jsut for the API (temporarily to allow testing)